server/src/language.js

```diff
@@ -157,16 +160,15 @@ class ProgramCacheItem {
         const program = this.getProgram(request.textDocument.uri);
 
         if (!program) {
-            return {
-                uri: "",
-                range: {
-                    start: { line: -1, character: -1},
-                    end: { line: -1, character: -1}
-                }
-            };
+            return null;
```

Comment:
This fixes two different bugs. The language server accepts "null" as meaning "no location"; -1 causes it to throw an exception.